mono_repo/CHANGELOG.md

```diff
@@ -232,7 +232,7 @@
       `mono_repo` during CI and run `mono_repo travis --validate` to ensure all
       files are up-to-date.
     - Respect the ordering of `stages`, if configured.
-    - Allow `stages` values to be just a string – allows defining an explicit
+    - Allow `stages` values to be just a string - allows defining an explicit
```

Comment:
Did you go em-dash crazy?

Reply:
Revert those changes – not helpful!

Reply:
You're using some non-standard dash chars! VS Code complained; I'll revert this part of the PR.